### PR TITLE
Reorganize left side menu

### PR DESCRIPTION
**Pull request by yurishkuro**
* Move some of the top-level pages into sub-pages
* Add summary of sub-pages at the top of the parent page (because left side menu is not  shown on mobile)

### CODE REVIEW

File: content/docs/1.13/deployment.md [hunk 1]
```diff
@@ -4,8 +4,11 @@ weight: 7
 children:
 - title: Operator for Kubernetes
   url: operator
+  navtitle: K8S Operator
```
Comment (jpkrohling) [inline content/docs/1.13/deployment.md]: `s/K8S/Kubernetes/`. If there's not much space, then just `s/K8S Operator/Kubernetes/`, as the operator is the preferred way to deploy on Kubernetes anyway.

File: content/docs/next-release/deployment.md [hunk 2]
```diff
@@ -3,6 +3,7 @@ title: Deployment
 weight: 7
 children:
 - title: Operator for Kubernetes
+  navtitle: K8S Operator
```
Comment (jpkrohling) [inline content/docs/next-release/deployment.md]: s/K8S/Kubernetes

**Comment by objectiser** [comment]
Two additional suggestions:
* Move `Client Libraries` top level below `Deployment` - in terms of ordering, it seems more appropriate to discuss deployment before talking about the ways of reporting data?
* Client Libraries page as a `Features` section which just points to its own child page. We should either move the content from that child page to this section, or remove this small section and just have the child page.

**Comment by yurishkuro** [comment]
@jpkrohling updated

**Pull request merged**